lib/decorateWithMetrics.js

```diff
@@ -6,6 +6,11 @@ import _ from 'underscore';
  */
 let stats = {};
 
+/* For some reason `performance.now()` does not start from `0` but a very large offset
+* like `508,080,000` see: https://github.com/facebook/react-native/issues/30069
+* Capturing an offset allows us to record start/ends times relative to app launch time */
+const APP_LAUNCH_TIME = performance.now();
```

Comment:
I've found this issue after I switched to the "new" debugging flow today: https://expensify.slack.com/archives/C01GTK53T8Q/p1626422647390900?thread_ts=1626297548.276300&cid=C01GTK53T8Q

Reply:
It's related to the "old" chrome debugger not being used